model_cards/bert-base-uncased-README.md

```diff
@@ -1,10 +1,231 @@
 ---
+language: en
```

Comment:
```suggestion
language: english
```

Reply:
(I haven't made the migration yet to 2-letter ISO-whatever codes)

Reply:
The template might need updating then ;-)